cmd/kind/get/nodes/nodes.go

```diff
@@ -51,6 +51,15 @@ func NewCommand() *cobra.Command {
 }
 
 func runE(flags *flagpole) error {
+	// Check if the cluster name exists
+	known, err := cluster.IsKnown(flags.Name)
```

Comment:
we shouldn't check, right?

Reply:
correct, but we might want to print a special message to stderr when there are no nodes

Reply:
(which can be done based on the length of nodes found below)